front/app/services/customPages.ts

```diff
@@ -174,17 +174,20 @@ export const FOOTER_PAGES: TFooterPage[] = [
 ];
 
 // Pages that exist in the static_pages database,
-// but do not have a corresponding navbar item.
+// but do not have a corresponding navbar item (with the exception of 'faq').
 // Their slugs and titles cannot be changed. Their
 // codes are the same as their slugs.
-type TFixedPage = TPolicyPage | 'proposals';
+type TFixedPage = TPolicyPage | 'faq' | 'proposals';
```

Comment:
The description above these actually says that their slug and titles cannot be changes. This makes sense, because their slugs are hard-coded in the codebase as well, and their titles have a function (e.g. 'Terms and conditions' should be called that). I guess let's also disable the slugs and title inputs for all fixed pages?

Reply:
You can't edit these `TPolicyPage` pages (not displayed on the Pages & menu page), but agree this needs some clarity. I'll see if I can still do that, without expanding the scope much.